### PR TITLE
AKS Baseline Multi Region - Customize manifests to pull images from the private ACR

### PR DESCRIPTION
**Pull request by magrande**

### CODE REVIEW

File: cluster-stamp.json [hunk 1]
```diff
@@ -710,7 +710,8 @@
             "name": "EnsureClusterIdentityHasRbacToPullAcr",
             "apiVersion": "2020-06-01",
             "dependsOn": [
-                "[resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', variables('clusterControlPlaneIdentityName'))]"
+                "[resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', variables('clusterControlPlaneIdentityName'))]",
```
Comment (ckittel) [inline cluster-stamp.json:713]: Do you still need this `dependsOn` entry?

**Review by ckittel — CHANGES_REQUESTED** [review]
Things I would have expected to see but don't

* A change to the Azure Policies to disallow these specific public container images
* A change to Azure Firewall rules to no longer support pulling these specific public container images
* A git add/commit/push for the change made to the kustomization yaml file

The stuff you have here looks good (only one tiny item below)

**Comment by ckittel** [comment]
Just one open item, and that's the `sed` command needs to be updated to impact both `kustomization.yaml` files now.

**Comment by magrande** [comment]
> Just one open item, and that's the `sed` command needs to be updated to impact both `kustomization.yaml` files now.

Fixed and tested !

**Review by ckittel — APPROVED** [review]
Cool, everything but traefik.  I'll add a backlog item to get that one in a future sprint.

**Pull request merged**